resources/aws-rds/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![doc = include_str!("../README.md")]
+#![allow(unused_imports)]
```

Comment:
The way the crate uses macro and features causes unused imports to be present. Not sure why CI didn't warn about it yet.

Reply:
Maybe we can address this if we're here. We can import the types by their full path: e.g. `serde::Serialize` instead of `Serialize`, and the same for the rest.